src/main/scala/temple/generate/database/PostgresGenerator.scala

```diff
@@ -61,8 +72,16 @@ object PostgresGenerator extends DatabaseGenerator {
             .mkString(",\n")
             .pipe(indent(_))
         s"CREATE TABLE $tableName (\n$stringColumns\n);"
-      case Read(tableName, columns) =>
+      case Read(tableName, columns, condition) =>
+        val stringColumns = columns.map(_.name).mkString(", ")
+        val stringCondition = condition.map(generateCondition) match {
+          case Some(conditionString) => List(s"WHERE $conditionString")
+          case None                  => List()
+        }
+        (s"SELECT $stringColumns FROM $tableName" +: stringCondition).mkString("", " ", ";")
+      case Insert(tableName, columns) =>
         val stringColumns = columns.map(_.name).mkString(", ")
-        s"SELECT $stringColumns FROM $tableName;"
+        val values        = (for (i <- 1.to(columns.length)) yield s"$$$i").toList.mkString(", ")
```

Comment:
This seems pretty convoluted, but I don’t know how it could be made nicer. Can the triple dollar be avoided? I guess the best alternatives would be `.format` or `'$' + i.toString` (we need some JS-level type coercion) 😉

I’d definitely infix the word `to`: it makes the arguments seem more equal.

Not too much of a change, but personally I think a map would be more legible than a for loop in a chain. While I don’t have a compiler to hand, `toList` doesn't seem like it should be necessary.

```scala
(1 to columns.length).map(i => "$$$i").mkString(", ")
```